ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -60,10 +64,30 @@ images:
     COPY . .
   from: src
   to: assisted-service-scripts
+- dockerfile_path: deploy/olm-catalog/bundle.Dockerfile
+  to: assisted-service-operator-bundle
 - dockerfile_literal: |
-    FROM base
-  from: ci-index
-  to: assisted-service-bundle
+    FROM quay.io/operator-framework/upstream-opm-builder:latest AS builder
+
+    ARG BUNDLE_IMGS=quay.io/edge-infrastructure/assisted-service-operator-bundle:latest
```

Comment:
Who published `quay.io/edge-infrastructure/assisted-service-operator-bundle:latest`?

Reply:
I guess it was produced by Jenkins previously (like the index).
Actually, I don't know at which moment the image in mirrored to quay, so we may have an issue if the bundle is deleted on quay and not pushed right after the build of `assisted-service-operator-bundle`.